src/app/tutorials/angular2-whatsapp-ionic-cli/index.ts

```diff
@@ -15,7 +15,7 @@ export const ANGULAR2_WHATSAPP_IONIC_CLI: TutorialDefinition = {
   },
   versions: {
     // Git revision => version definition object
-    'master': V2_0_0,
+    'master': V3_0_0,
```

Comment:
My goal was that for every new version of the tutorial we create a branch.
That was at least my goal when designing the versioning of Tortilla with @DAB0mB 
@DAB0mB , isn't that the way it works now?  I don't want to lose previous versions when we release new ones